tests/unit/test_core/test_auth.py

```diff
@@ -131,13 +132,19 @@ async def test_03_create_user_request_bad(
     ):
         """Test auth create user endpoint."""
         auth.controller.loop = event_loop
-        exc = ValidationError('', BaseModel)
+        self.user_data["email"] = "wrong_email"
+        exc = ValidationError(
```

Comment:
@Alopalao, for completeness, could you not mock the `create_user.side_effect.ValidatioError` side effect here? In summary, the unit test guarantee that I'm looking for on test case here, is ensuring that `pydantic` internally error_list with `"loc"` will still work. As mentioned, `pydantic` probably won't break compatibility here, but if it ever does, then this test will also catch it. See what I mean? 

The test case itself is looking great though. I'll leave the PR pre-approved since this is just unit test implementation minor detail, once it's addressed, let's ship it.

Reply:
`Auth.get_user_controller` is being replaced by MagicMock(), Any dictionary sent to the request will result in 201. I can a BadRequest only by sending a non-dictionary to `(json=)`.

Reply:
Indeed @Alopalao, let's keep this one this way then. I was thinking of instanting a new `UserController` and not mock `create_user`, something like `UserController(get_mongo=lambda: MagicMock())`, but it'll require way more work mocking other methods. So, instead to simplify and still cover that a non-mocked `ValidationError` is covered, can you adapt `test_error_msg` when trying to instantiate a `UserDoc`, and catching it and passing to `error_msg` and asserting the expected message? That way it won't cover in a request context, but a least we won't be mocking `ValidationError` entirely, just so if `pydantic` ever changes that test case will still catch it.

Reply:
Got it, changed `test_error_msg`